diagnostic.md

```diff
@@ -84,7 +103,9 @@ div {
 ```
 
 <!-- your answer starts here -->
+specialDiv will have a font family of sans-serif and a color of yellow.
```

Comment:
the font family will be cursive because inline style is more specific than class styles